emva1288/process/results.py

```diff
@@ -930,7 +929,7 @@ def histogram_DSNU(self):
         h = routines.Histogram1288(y, self._histogram_Qmax)
         # Rescale the bins, this is due to upscaling the average image to have
         # only integers
-        scale = self.spatial['L_dark'] * 25.
```

Comment:
why remove the 25 factor?

Reply:
Because the horizontal scale of the DSNU histogram is currently wrong by a factor of 25, unless I'm missing something... What was the motivation to put it there in the first place? The histogram should be calculated by summing the dark images and then dividing the horizontal scale by the number of images (L_dark), yielding the histogram of the averaged image. I don't see where a factor of 25 should pop up.

Reply:
I think @thomasdob  is right, @JF1008  can you confirm with your images?

Reply:
I can confirm this, I'll link the notebook tomorrow morning!